README.md

```diff
@@ -182,6 +182,14 @@ Typically, it is easier to get to work with `pdflatex` than PSTricks is.
 - [A very minimal introduction to TikZ (PDF)](https://cremeronline.com/LaTeX/minimaltikz.pdf) - A short introductory document to the world of TikZ, written by Jacques Crémer.
 - [PetarV-/TikZ](https://github.com/PetarV-/TikZ) - A collection of publication-ready PGF/TikZ figures by Petar Veličković.
 
+### Source Code
+
+- [minted](https://www.ctan.org/pkg/minted) - The minted package uses [pygments](http://pygments.org/) to generate the listings. In that way, LaTeX is able to format more than 300 languages and other text formats.
```

Comment:
I'll change it to "programming and markup languages", because XML is no programming language per se, although there are programming languages in XML (such as [BPEL](https://en.wikipedia.org/wiki/Business_Process_Execution_Language))